server/resp/src/main/java/org/infinispan/server/resp/commands/connection/MemoryEntrySizeUtils.java

```diff
@@ -16,11 +14,6 @@ public static long calculateSize(byte[] key, InternalCacheEntry<byte[], Object>
    }
 
    static long internalCalculateSize(byte[] key, Object value) {
-      try {
-         return sizeof.deepSizeOf(key) + sizeof.deepSizeOf(value);
-      } catch (Exception ex) {
```

Comment:
Are we not worried about the exception that was thrown? Or I guess this is just to reproduce the exception?

Reply:
yeah, unfortunately the current implementation swallows the original exception